src/index.tsx

```diff
@@ -13,6 +13,12 @@ const LazyConfig = React.lazy(async () => import('./pages/config.jsx'))
 const LazyHelperUi = React.lazy(async () => import('./pages/helper-ui.jsx'))
 const LazyRedirectPage = React.lazy(async () => import('./pages/redirect-page.jsx'))
 const LazyInterstitial = React.lazy(async () => import('./pages/redirects-interstitial.jsx'))
+const LazyServiceWorkerErrorPage = React.lazy(async () => import('./pages/errors/no-service-worker.jsx'))
+
+let ErrorPage: null | React.LazyExoticComponent<() => React.JSX.Element> = null
+if (navigator.serviceWorker == null) {
```

Comment:
```suggestion
if ('serviceWorker' in navigator) {
```

Reply:
idk that the `in` keyword gives us any benefit for this use case.

Reply:
mostly defensive programming: easier to read, and if updated, linter in the future asks someone to "fix" `==` to `===` we are exposed to bug (seen it before).

Reply:
eh, I can get in line with that reasoning.